defs/bitcoin/primecoin.json

```diff
@@ -0,0 +1,38 @@
+{
+  "coin_name": "Primecoin",
+  "coin_shortcut": "XPM",
+  "coin_label": "Primecoin",
+  "website": "https://primecoin.io",
+  "github": "https://github.com/primecoin/primecoin",
+  "maintainer": "James Skrowvedeht <james@thedevworks.io>",
+  "curve_name": "secp256k1",
+  "address_type": 23,
+  "address_type_p2sh": 83,
+  "maxfee_kb": 1000000,
+  "minfee_kb": 1000,
+  "signed_message_header": "Primecoin Signed Message:\n",
+  "hash_genesis_block": "963d17ba4dc753138078a2f56afb3af9674e2546822badff26837db9a0152106",
+  "xprv_magic": 76066276,
+  "xpub_magic": 76067358,
+  "xpub_magic_segwit_p2sh": null,
+  "xpub_magic_segwit_native": null,
+  "bech32_prefix": null,
+  "cashaddr_prefix": null,
+  "slip44": 24,
+  "segwit": false,
+  "decred": false,
+  "fork_id": null,
+  "force_bip143": false,
+  "bip115": false,
+  "default_fee_b": {
+    "Normal": 10
+  },
+  "dust_limit": 546,
+  "blocktime_seconds": 60,
+  "uri_prefix": "primecoin",
+  "min_address_length": 35,
```

Comment:
How did you come up with values for min_address_length and max_address_length? I picked a random address from the explorer and it was 34 characters long. Please adjust the values to correct ones.

Reply:
Corrected! Apologies for the oversight.. I shouldn't do PR's at 2am! ;)